src/config-schema.ts

```diff
@@ -37,9 +37,10 @@ export const configSchema = {
       "problem",
     ],
   },
-  patientUuidConfig: {
+  patientUuid: {
     _type: "String",
-    _default: "b280078a-c0ce-443b-9997-3c66c63ec2f8",
-    _description: "Patient UUID used to render form preview",
+    _default: "88f1032f-adae-4ef2-9025-2c40b71dd897",
```

Comment:
Is the patient with this UUID always there even with the various server outages and restarts?

Reply:
This one has proven to be a bit more resilient than most.